aten/src/ATen/native/BinaryOps.cpp

```diff
@@ -164,6 +164,10 @@ Tensor add(const Tensor& self, Scalar other, Scalar alpha) {
   return native::add(self, wrapped_scalar_tensor(other), alpha);
 }
 
+Tensor add(Scalar other, const Tensor& self, Scalar alpha) {
+  return native::add(self, other, alpha);
```

Comment:
this doesn't look right.

In the "normal" version, this op is:
`add(p1, p2,  alpha) -> p1 + p2*alpha`

but you are computing:
`add(p1, p2, alpha) -> p2 + p1*alpha`

though renaming all the variables makes this confusing (it doesn't matter in C++).

Reply:
Good point! I'll fix this and add a test.